tools/scripts-config-react-webpack/config/utils/cache.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const path = require('path');
+const crypto = require('crypto');
+
+function getLockFilePath(cwd = process.cwd()) {
+	const yarnlockPath = path.join(cwd, 'yarn.lock');
+	const pkglockPath = path.join(cwd, 'package-lock.json');
+	if (fs.existsSync(yarnlockPath)) {
+		return yarnlockPath;
+	} else if (fs.existsSync(pkglockPath)) {
+		return pkglockPath;
+	}
+	try {
+		return getLockFilePath(path.join(cwd, '..'));
```

Comment:
It's recursive but without a proper stop condition

Reply:
I have changed with parentPath === cwd because:
```
> path.join('/', '..')
'/'
```

Reply:
fixed based on 
```
> path.join('/', '..')
'/'
```
so parent === cwd